core/trino-main/src/main/java/io/trino/eventlistener/EventListenerManager.java

```diff
@@ -14,158 +14,23 @@
 package io.trino.eventlistener;
```

Comment:
I am not sure if this the proper abstraction you need. Notice that you don't care about methods like `addEventListener` or `load*`. You care only about events handling. So maybe you could use `EventListener` for statistics collection, where one `EventListener` would be a wrapper on a list of `EventListener`s that is calling them in a loop? WDYT?